requirements.txt

```diff
@@ -1 +1,6 @@
-GeoIP
\ No newline at end of file
+GeoIP
+web.py
+  cheetah
+lxml
+psycopg2
+redis
```

Comment:
I wondered if we can freeze versions there as this dependencies can be different for different envs that have potential danger.

Reply:
`pip install GeoIP` fails on my machine.  Please see #11

Reply:
@cclauss, looks you have issue with system dependencies: https://stackoverflow.com/a/25963432/880326

Reply:
@cclauss or maybe you can convince Maxmind to release wheels to legacy GeoIP https://github.com/maxmind/geoip-api-python/issues/28

Reply:
@tbicr Suggestion fixed #11